packages/blade/src/components/Icons/iconMap.ts

```diff
@@ -84,6 +84,7 @@ const iconMap: Record<string, IconComponent> = {
   RotateCounterClockWiseIcon: RotateCounterClockWiseIconComponent,
   TrendingUpIcon: TrendingUpIconComponent,
   TrendingDownIcon: TrendingDownIconComponent,
+  MailIcon: MailIconComponent,
```

Comment:
Can we have it at the right alphabetical position? Easier for folks to navigate. Check for other files too.